pkg/ingester/ingester_v2.go

```diff
@@ -1216,8 +1217,30 @@ func (i *Ingester) compactionLoop(ctx context.Context) error {
 	return nil
 }
 
+// compactHead compact the Head block at 2h intervals,
+// avoiding a single huge block.
+func compactHead(userDB *userTSDB) error {
+	// The Compact() will produces multiple 2h blocks if Head has a lot of data.
```

Comment:
Do not hardcode "2h" in the comment because it depends on the configured block range period